Rakefile

```diff
@@ -7,8 +7,8 @@ require_relative './lib/pmdtester/parsers/options.rb'
 
 gem 'hoe'
 require 'hoe'
+
 Hoe.plugin :bundler
-Hoe.plugin :gemspec
```

Comment:
Why don't we need hoe-gemspec anymore? I thought, we would require this for creating a ruby gem...

Reply:
We create a gemspec file by [hoe:spec](https://github.com/pmd/pmd-regression-tester/blob/bfe46f7d50f712830ece6ee8b63a7c080b1a6c76/Rakefile#L49) task now.

Reply:
:+1: